backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceReport.java

```diff
@@ -183,6 +184,23 @@ public BacktraceReport(
         if (this.exceptionTypeReport && exception != null) {
             this.classifier = exception.getClass().getCanonicalName();
         }
+        this.setErrorTypeAttribute();
+    }
+
+    /**
+     * Sets error.type attribute depends on the type of the report
+     */
+    private void setErrorTypeAttribute() {
```

Comment:
probably it will be better to name it `setDefaultErrorTypeAttribute` or keep as it is but add parameter with type

Reply:
yeah, it's better. thx